cmd/run_test.go

```diff
@@ -125,3 +132,48 @@ func TestHandleSummaryResultError(t *testing.T) {
 	assertEqual(t, "file summary 1", files[filePath1])
 	assertEqual(t, "file summary 2", files[filePath2])
 }
+
+func TestAbortTest(t *testing.T) { //nolint: tparallel
```

Comment:
Is this `nolint` required :thinking:

Reply:
I think you can just add `t.Parallel` to the ones below? Or just split them in two separate tests?

Reply:
Sorry, missed this originally.

To answer your first question: yes, it's needed, otherwise `tparallel` fails with `TestAbortTest's subtests should call t.Parallel`. And both `tparallel` and `paralleltest` are enabled by default... It's confusing, I know :disappointed: Maybe we should disable `tparallel` globally?

We can't add `t.Parallel()` to either test because of several data races... And I prefer having them as subtests in this case. :)